src/main/groovy/com/morpheusdata/commvault/backup/CommvaultBackupTypeProvider.groovy

```diff
@@ -21,9 +20,15 @@ class CommvaultBackupTypeProvider extends AbstractBackupTypeProvider {
 
```

Comment:
This should really be an abstract class as it isn't meant to be used directly.

Reply:
@ddeyoung changes are made as suggested